src/display/annotation_helper.js

```diff
@@ -277,6 +316,8 @@ var AnnotationUtils = (function AnnotationUtilsClosure() {
         return getHtmlElementForTextAnnotation(data);
       case AnnotationType.LINK:
         return getHtmlElementForLinkAnnotation(data);
+      case AnnotationType.VIDEO:
```

Comment:
After rebasing this patch, change this line to:

<pre>
case AnnotationType.MOVIE:
case AnnotationType.SCREEN:
  return ...
</pre>